MobileNav/ViewControllers/Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,158 @@
+        settingsTableView.tableFooterView = UIView()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
+        updateButtonProxyState(proxyState)
+    }
+
+    private func presentAlertController() {
+        let alert = UIAlertController(title: "Incomplete Form", message: "You must select a render type and stream type before proceeding", preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "OK", style: .default)
+        alert.addAction(okAction)
+        self.present(alert, animated: true)
+    }
+
+    private func startSDL(with renderType:RenderType, streamType:StreamType) {
+
+        // Save selected options to user defaults
+        AppUserDefaults.shared.renderType = renderType
+        AppUserDefaults.shared.streamType = streamType
+
+        // Connect SDL with selected settings
+        let streamSettings = StreamSettings(renderType: renderType, streamType:streamType)
+        ProxyManager.sharedManager.connect(with: SDLAppConstants.connectionType, streamSettings: streamSettings)
+    }
+}
+
+// MARK: - Update Button State
+
+extension SettingsViewController {
+    func updateButtonProxyState(_ newState: ProxyState) {
+        var newColor: UIColor? = nil
+        var newTitle: String? = nil
+
+        switch newState {
+        case .stopped:
+            newColor = UIColor.systemGreen
+            newTitle = "Start".uppercased()
+        case .searching:
+            newColor = UIColor.systemBlue
+            newTitle = "Stop Searching".uppercased()
+        case .connected:
+            newColor = UIColor.systemRed
+            newTitle = "Disconnect".uppercased()
+        }
+
+        if (newColor != nil) || (newTitle != nil) {
+            DispatchQueue.main.async(execute: {[weak self]() -> Void in
+                self?.startButton.backgroundColor = newColor
+                self?.startButton.setTitle(newTitle, for: .normal)
+            })
+        }
+    }
+}
+
+// MARK: - Selecting Options
+
+extension SettingsViewController: SettingOptionsViewControllerDelegate {
+    func optionSelected(option: Int) {
+        if settingOptions == RenderType.allCases.map({ $0.description }) {
+            selectedRenderType = RenderType(rawValue: option)
+        } else {
+            selectedStreamType = StreamType(rawValue: option)
+        }
+        settingsTableView.reloadData()
+    }
+}
+
+// MARK: - UITableViewDelegate & UITableViewDataSource
+
+extension SettingsViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return SettingsOptions.allCases.count
+    }
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "reuseIdentifier", for: indexPath)
```

Comment:
Can you set the default "render" and "stream" types in the table view cell (like how they get set after you select an options from the sub-menu). Right now it looks like nothing is selected.